src/NmeaParser.Shared/Nmea/Gps/GPRMC.cs

```diff
@@ -39,7 +39,7 @@ protected override void OnLoadMessage(string[] message)
 			if (message == null || message.Length < 11)
 				throw new ArgumentException("Invalid GPRMC", "message"); 
 			
-			if (message[8].Length == 6 && message[0].Length == 6)
+			if (message[8].Length == 6 && message[0].Length >= 6)
 			{
```

Comment:
According to U Blox specification it is:
https://www.u-blox.com/images/downloads/Product_Docs/u-blox6_ReceiverDescriptionProtocolSpec_%28GPS.G6-SW-10018%29.pdf

the specified format is hhmmss.ss . Not sure if those last ss supposed to
be hundreths of a second because at least my device outputs zeroes after
the dot.


tiistai 24. helmikuuta 2015 Morten Nielsen <notifications@github.com>
kirjoitti:

> In src/NmeaParser.Shared/Nmea/Gps/GPRMC.cs
> <https://github.com/dotMorten/NmeaParser/pull/5#discussion_r25288557>:
>
> > @@ -39,7 +39,7 @@ protected override void OnLoadMessage(string[] message)
> >  			if (message == null || message.Length < 11)
> >  				throw new ArgumentException("Invalid GPRMC", "message");
> >  			
> > -			if (message[8].Length == 6 && message[0].Length == 6)
> > +			if (message[8].Length == 6 && message[0].Length >= 6)
> >  			{
>
> Why would the length ever be greater than 6? That doesn't seem to be part
> of the spec?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dotMorten/NmeaParser/pull/5/files#r25288557>.
>